assembly/pom.xml

```diff
@@ -89,7 +89,18 @@
           <artifactSet>
             <includes>
               <include>*:*</include>
+              <!--use the following line if we only want carbondata related classes-->
+              <!--<include>org.apache.carbondata:*</include>-->
```

Comment:
FYI: If we use this line of code, the assembly jar will only contain carbondata related classes and the final size of assembly-jar will be about 7MB.

But there will be some problems: If carbon use an artifact that is not provided by spark, it will cause runtime error. So we'd better to keep the classes and only remove those are known to be provided by spark.